scripts/build

```
@@ -8,6 +8,6 @@ fi
 
 set -x
 
-${PREFIX}python setup.py sdist bdist_wheel
+${PREFIX}python -m build --sdist --wheel
```

Seems like this is the default for `python -m build`?

```suggestion
${PREFIX}python -m build
```

https://pypa-build.readthedocs.io/en/latest/index.html